tests/etl/test_dsv_to_table.py

```diff
@@ -53,6 +53,65 @@ def test_pipeline_with_comtrade_csv(self, app_with_db):
         ]
         assert rows_equal_table(app_with_db.dbi, expected_rows, pipeline._l0_table, pipeline)
 
+    def test_pipeline_with_comtrade_csv_with_duplcate_rows(self, app_with_db):
+        pipeline = DSVToTablePipeline(
+            app_with_db.dbi,
+            organisation='comtrade',
+            dataset='country_code_and_iso',
+            data_column_types=[
+                ('ctyCode', 'int'),
+                ('cty Name English', 'text'),
+                ('cty Fullname English', 'text'),
+                ('Cty Abbreviation', 'text'),
+                ('Cty Comments', 'text'),
+                ('ISO2-digit Alpha', 'text'),
+                ('ISO3-digit Alpha', 'text'),
+                ('Start Valid Year', 'text'),
+                ('End Valid Year', 'text'),
+            ],
+        )
+        fi = FileInfo.from_path('tests/fixtures/generic_dsv/country_list dups.csv')
```

Comment:
Again super tiny, but generally I think it's nice to not have spaces in filenames (file itself will need updating too)?

```suggestion
        fi = FileInfo.from_path('tests/fixtures/generic_dsv/country_list_dups.csv')
```

Reply:
I was wondering why my test was failing initially with `file not found error`. Good old `_`